Introductory Tests/Equations_of_motion.py

```diff
@@ -0,0 +1,59 @@
+# General
+import numpy as np
+
+# For computing acceleration and potential
+import polyhedral_gravity as model
+
+
+class Equations_of_motion:
+    """
+    Defining the celestial body of interest as a mesh object, as well as providing methods 
+    for computing the satellite's acceleration and motion with respect to the celestial
+    body and its current position expressed in three dimensions.
+    """
+
+    def __init__(self, mesh_vertices, mesh_faces, body_density):
+        """
+        Construtor for defining the celestial body as a mesh-based object.
+
+        Args:
+            mesh_vertices (_array_): Array containing all vertices of the mesh.
```

Comment:
is there a reason you are putting `_` around the types? have never seen this before

Reply:
(it doesn't matter much in case you wanna keep, just rare to me)